javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -1083,17 +1064,51 @@ private PathNode getASuccessor(PathNode pnd) {
 }
 
 private PathNode getASuccessorIfHidden(PathNode nd) {
-  nd.isHidden() and
+  nd.(MidPathNode).isHidden() and
   result = getASuccessor(nd)
 }
 
+/**
```

Comment:
Is the diff bad, or did you include two qldoc starters here?
```
/**
/**
```

Reply:
The latter; will fix.